src/components/Forms/InlineValuePopover.jsx

```diff
@@ -1,27 +1,39 @@
 import clsx from "clsx";
-import React from "react";
-import { Popover } from "../Popover/Popover";
+import React, { useRef } from "react";
 import { DownArrowIcon } from "../..";
+import { Popover } from "../Popover/Popover";
 
 export const InlineValuePopover = ({
     text,
     isOpen = false,
-    showArrow = false,
+    showArrow = true,
     onClick,
     onClickOutside,
     classNames,
     children,
     ...rest
 }) => {
+    const ref = useRef();
+    const handleClick = (e) => {
+        setTimeout(() => {
+            // This may technically belong in Seller app, but the most common use case for this component here is
+            // to use inputs, so we'll provide some helpers for that
+            ref?.current.querySelector("select,textarea,input")?.focus();
+        }, 0);
+        onClick?.(e);
+    };
+
     return (
         <Popover visible={isOpen} onClickOutside={onClickOutside} {...rest}>
-            <span className={clsx("cursor-pointer", classNames?.text)} onClick={onClick}>
-                <span className="border-b border-primary font-bold">{text}</span>
+            <span className={clsx("cursor-pointer", classNames?.text)} onClick={handleClick}>
```

Comment:
can we add wrap styles to make arrow text and arrow to stay in one line, instead of arrow moving to another line.

Reply:
Can you give me an example of this? I don't how to reproduce the issue, so don't know what to fix

Reply:
Ok, I think I figured it out. Let me know if this is it:

Before:

![CleanShot 2022-09-12 at 08 33 04@2x](https://user-images.githubusercontent.com/65708/189566353-88fabf35-4a8e-492b-829e-f9af18213229.png)

After: 

![CleanShot 2022-09-12 at 08 33 31@2x](https://user-images.githubusercontent.com/65708/189566376-50c97d96-c040-4003-ab19-e1ca02a71b7a.png)

Ignore the blue background, that's just to illustrate the size of the container (`w-10`)

Reply:
yes, this is the error